crates/dc_bundle/src/definition/element.rs

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use std::fmt;
+use std::fmt::{Debug, Display, Formatter};
+include!(concat!(env!("OUT_DIR"), "/designcompose.definition.element.rs"));
+
+impl Display for FontStyle {
+    fn fmt(&self, f: &mut Formatter) -> fmt::Result {
+        Debug::fmt(self, f)
+    }
+}
+impl FontFeature {
+    pub fn new(tag: String) -> Self {
+        FontFeature { tag, enabled: true }
+    }
+}
```

Comment:
Is the plan to add other `impl` functions for types in `legacy_definition/element/` as we convert new types?

Reply:
You mean `definition/element/`?

Yeah. If it starts getting unwieldly then we can split them into seperate submodules of element.rs, like how they are in legacy_definition.